guardian/security-hq
# Add cloudformation:ListStacks permission to SecurityHQRole

> **Warning**
> This change must be rolled out by March 27, 2023

## What does this change?

This is required in order to address an upcoming AWS change that requires that customers have the cloudformation:ListStacks permission when calling DescribeStacks without a target stack.

> We require your action to update IAM Policies in your account by March 27, 2023 to avoid disruption to your DescribeStacks API calls.
> 
> AWS CloudFormation now requires that customers have the cloudformation:ListStacks permission when calling DescribeStacks[1] without a target stack. This is in addition to the cloudformation:DescribeStacks permission that is already a requirement. You will need to add the cloudformation:ListStacks permission by March 27, 2023, on any calling principles using the DescribeStacks API without a target. CloudFormation has identified principals associated with your account that do not have ListStacks permissions making DescribeStacks calls without a target. 

## How to deploy

This change requires manual deployment to the associated StackSet in the root account, and will need to be applied to all accounts in our organisation.

## How to test

The StackSet deployment from the root account will succeed and you will be able to visit any other organisation account and see the associated changes in the relevant CloudFormation stack controlled by the StackSet. 

## How can we measure success?

SecurityHQ continues to work after March 27, 2023.
